src/Symfony/Component/Routing/composer.json

```diff
@@ -18,6 +18,7 @@
     "require": {
         "php": "^7.2.5",
         "symfony/deprecation-contracts": "^2.1",
+        "symfony/polyfill-php73": "^1.9",
```

Comment:
Needed to use `array_key_first` in `XmlFileLoader`: https://github.com/symfony/symfony/pull/36187/files#diff-d759b630af7ee68956cf54745e871ee5R317